src/Algebra.jl

```diff
@@ -1,4 +1,22 @@
 
+# To be added in Gridap?
```

Comment:
For the moment we can leve it here to avoid the burden of moving it to Gridap registering a new version etc

But I agree that at some point this can be moved